astropy/stats/bayesian_blocks.py

```diff
@@ -496,11 +509,3 @@ def validate_input(self, t, x, sigma):
             raise ValueError("x must be specified for point measures")
         return super(PointMeasures, self).validate_input(t, x, sigma)
 
-    def prior(self, N, Ntot):
-        if self.gamma is not None:
-            return self.gamma_prior(N, Ntot)
-        elif self.p0 is not None:
-            return self.p0_prior(N, Ntot)
-        else:
-            # eq. at end of sec 3.3 in Scargle 2012
-            return 1.32 + 0.577 * np.log10(N)
```

Comment:
Where did this prior form go? If I recall correctly, this is empirically calibrated for point measures, so we should provide this form somewhere in the class.